install.sh

```diff
@@ -65,7 +65,7 @@ python scripts/repo_manager.py
     --test-bench-generator        master                                        \
     --test-bench-generator-remote github.com/StanfordAHA/TestBenchGenerator.git \
                                                                                 \
-    --cgra-pnr                    master                                        \
```

Comment:
looks like you changed CGRAFlow to point to cgra-pnr *dev* branch instead of master...
instead, why not just pull cgra-pnr dev -> cgra-pnr master?

Reply:
This change is so we can test that the cgra-pnr dev branch works on travis before it is merged to master. Since it works, we should merge these changes and point it back to master after cgra-pnr dev has been merged to master.

Reply:
sounds good.